src/helm/benchmark/scenarios/cleva_scenario.py

```diff
@@ -0,0 +1,1608 @@
+                    for _k in data[_name]:
+                        # Only perform the subsitution if the keys match
+                        if _k == k:
+                            assert isinstance(
+                                data[_name][_k], str
+                            ), "We only support mapping data with type `Dict[str, str]`"
+                            data[_name][_k] = d[data[_name][_k]]
+                # If the value is a list, then look into its entries
+                elif isinstance(data[_name], list):
+                    assert len(data[_name]) > 0, f"The length of {_name} must be larger than 0."
+                    # We use the first element for type checking, assuming all entries are of the same type
+                    if isinstance(data[_name][0], int):
+                        pass
+                    elif isinstance(data[_name][0], str):
+                        # If the entry is a string and the key matches, we directly map all entries
+                        if _name == k:
+                            data[_name] = [d[c] for c in data[_name]]
+                    # If the entry is a dict, we look into its key-value pairs
+                    elif isinstance(data[_name][0], dict):
+                        for item in data[_name]:
+                            for _k in item:
+                                # Only perform the subsitution if the keys match
+                                if _k == k:
+                                    assert isinstance(
+                                        item[_k], str
+                                    ), "We only support mapping data with type `List[Dict[str, str]]`"
+                                    item[_k] = d[item[_k]]
+                    else:
+                        raise NotImplementedError(
+                            "We only support mapping data with type `List[str]` or `List[Dict[str, str]]`"
+                        )
+                else:
+                    raise NotImplementedError("We only support mapping data with type `list` or `str`")
+
+    @staticmethod
+    def index_mapping(idx: int, option: str) -> str:
+        """This function defines how to index a list of values according to the given option."""
+        if option is None:
+            return ""
+        elif option == "number":
+            return f"{idx + 1}"
+        elif option == "upper":
+            return chr(ord("A") + idx)
+        elif option == "lower":
+            return chr(ord("a") + idx)
+        else:
+            raise NotImplementedError(f"Unknown option {option}")
+
+
+class CLEVAScenario(Scenario):
+    """
+    Scenario for CLEVA benchmark (https://arxiv.org/pdf/2308.04813.pdf).
+    """
+
+    name = "cleva"
+    splits: Dict[str, str] = {
+        "train": TRAIN_SPLIT,
+        "test": TEST_SPLIT,
+    }
+
+    def __init__(
+        self,
+        version: str,
+        subtask: str,
+        prompt_template: Dict[str, Any],
```

Comment:
Replace ` prompt_template: Dict[str, Any]` with `prompt_id: int` and then inside the `__init__`() body do:

```python
self.prompt_template, _ = CLEVAScenario.get_prompt_setting(task, subtask, version, prompt_id)
```